Plugins/Flow.Launcher.Plugin.Sys/Main.cs

```diff
@@ -192,11 +192,161 @@ private List<Result> Commands()
+                {
+                    Title = "Documents",
+                    SubTitle = context.API.GetTranslation("flowlauncher_plugin_sys_documents"),
+                    IcoPath = "Images\\documents.png",
+                    Glyph = new GlyphInfo (FontFamily:"/Resources/#Segoe Fluent Icons", Glyph:"\xe8a5"),
+                    Action = c =>
+                    {
+                        {
+                            System.Diagnostics.Process.Start("explorer.exe", "shell:Personal");
+                        }
+
+                        return true;
+                    }
+                },
+                 new Result
+                {
+                    Title = "Music",
+                    SubTitle = context.API.GetTranslation("flowlauncher_plugin_sys_music"),
+                    IcoPath = "Images\\music.png",
+                    Glyph = new GlyphInfo (FontFamily:"/Resources/#Segoe Fluent Icons", Glyph:"\xe93c"),
+                    Action = c =>
+                    {
+                        {
+                            System.Diagnostics.Process.Start("explorer.exe", "shell:My Music");
+                        }
+
+                        return true;
+                    }
+                },
+                 new Result
+                {
+                    Title = "Index Option",
+                    SubTitle = context.API.GetTranslation("flowlauncher_plugin_sys_indexoption"),
+                    IcoPath = "Images\\indexoption.png",
+                    Glyph = new GlyphInfo (FontFamily:"/Resources/#Segoe Fluent Icons", Glyph:"\xe773"),
+                    Action = c =>
+                    {
+                        {
+                            System.Diagnostics.Process.Start("control.exe", "srchadmin.dll");
+                        }
+
+                        return true;
+                    }
+                },
+                new Result
+                {
+                    Title = "Downloads",
+                    SubTitle = context.API.GetTranslation("flowlauncher_plugin_sys_downloads"),
+                    IcoPath = "Images\\download.png",
+                    Glyph = new GlyphInfo (FontFamily:"/Resources/#Segoe Fluent Icons", Glyph:"\xe7f4"),
+                    Action = c =>
+                    {
+                        {
+                            System.Diagnostics.Process.Start("explorer.exe", "shell:downloads");
+                        }
+
+                        return true;
+                    }
+                },
+                new Result
                 {
                     Title = "Empty Recycle Bin",
                     SubTitle = context.API.GetTranslation("flowlauncher_plugin_sys_emptyrecyclebin"),
                     IcoPath = "Images\\recyclebin.png",
-                    Glyph = new GlyphInfo (FontFamily:"/Resources/#Segoe Fluent Icons", Glyph:"\xe74d"),
+                    Glyph = new GlyphInfo (FontFamily:"/Resources/#Segoe Fluent Icons", Glyph:"\xe896"),
```

Comment:
Is this intended? `e896` is the download icon? 
![image](https://user-images.githubusercontent.com/45326534/189489096-b61c5f95-19c5-47b2-8eb6-76fb32256fe0.png)

Reply:
mistake. Fixed e74d from e896.